packages/es-components/src/components/containers/menu/MenuPanel.tsx

```diff
@@ -59,15 +62,15 @@ const MenuPanel = React.forwardRef<HTMLDivElement, MenuPanelProps>(
 
       function onEscape({ key }: { key: string }) {
         if (key === 'Escape') {
-          onClose();
+          (propsRef.current.onClose || noop)();
```

Comment:
Do we still need to follow this pattern? Now that we've moved to TS `onClose?.()` is possible and more succinct.

Reply:
Makes sense! I'll make the update!

Reply:
(I honestly forgot that pattern was available. 😆)